### PR TITLE
NXP-25284: Change collection member logic to nuxeo-filter

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

**Comment by afjusto** [comment]
👍

**Comment by guirenard** [comment]
:+1:

File: elements/nuxeo-web-ui-bundle.html [hunk 1]
```diff
@@ -241,7 +241,11 @@
 
 <nuxeo-slot-content name="clipboardToggleDocumentAction" slot="DOCUMENT_ACTIONS" order="60">
   <template>
-    <nuxeo-clipboard-toggle-button document="[[document]]" clipboard="[[clipboard]]"></nuxeo-clipboard-toggle-button>
+    <nuxeo-filter document="[[document]]" expression="document.facets.indexOf('NotCollectionMember') === -1">
```
[the remark this answers is not in the document]
Comment (afjusto) [reply elements/nuxeo-web-ui-bundle.html]: or even use directly in the expression https://github.com/nuxeo/nuxeo-ui-elements/blob/master/nuxeo-filters-behavior.html#L49 if possible

**Comment by Gabez0r** [comment]
Indeed, the `isVersion` check would make sense in the filter, because there's nothing preventing that action for working for versions. However, right now, the clipboard only allows us to move or copy documents, which doesn't make sense versions. I have mixed feelings.

**Comment by nmpcunha** [comment]
After some investigation and testing, currently there is no real purpose of "allowing" versions to be added to the clipboard because it just duplicates the document itself.
Maybe with some new clipboard features and action it may make sense to use versions on clipboard and then we can change the filter again.

**Comment by Gabez0r** [comment]
👍

**Comment by guirenard** [comment]
:+1:

**Pull request merged**